stdlib/collections/__init__.pyi

```diff
@@ -129,8 +131,11 @@ class UserString(Sequence[UserString]):
     def __iter__(self: Self) -> Iterator[Self]: ...
     def __reversed__(self: Self) -> Iterator[Self]: ...
     def __add__(self: Self, other: object) -> Self: ...
+    def __radd__(self: Self, other: object) -> Self: ...
     def __mul__(self: Self, n: int) -> Self: ...
+    def __rmul__(self: Self, n: int) -> Self: ...
     def __mod__(self: Self, args: Any) -> Self: ...
+    def __rmod__(self: Self, template: object) -> Self: ...
```

Comment:
This one is actually broken:

```
In [12]: [] % collections.UserString("x")

Traceback (most recent call last):
  File "/main_instance_shell/jelle/venv/lib/python3.6/site-packages/IPython/core/interactiveshell.py", line 3343, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-12-0ca5ac41c5a2>", line 1, in <module>
    [] % collections.UserString("x")
  File "/main_instance_shell/jelle/venv/lib/python3.6/collections/__init__.py", line 1162, in __rmod__
    return self.__class__(format % args)
NameError: name 'args' is not defined
```

It should error but not with a NameError.

Reply:
Go ahead! I think this method just shouldn't exist. It's presumably there to support %-formatting, but I don't think a reflected method makes sense for that.

Reply:
Wait, I can't reproduce the `NameError`. This is what I get on my machine:

```python
>>> from collections import UserString
>>> [] % UserString('foo')
'[]'
>>> [] % UserString('x')
'[]'
>>>
```

Reply:
Ah, it got fixed in https://bugs.python.org/issue25652. That's what I get for trying things out on 3.6. I admit I don't understand the reasoning they ended up with for keeping the method.

Reply:
```
In [17]: collections.UserString("x") + collections.UserList(["y"])
Out[17]: "x['y']"

In [18]: collections.UserList(["y"]) + collections.UserString("x")
Out[18]: ['y', 'x']
```

Reply:
Truly a nightmare of what Python could have been